remote_api/client.go

```diff
@@ -51,19 +57,31 @@ func NewRemoteContext(host string, client *http.Client) (context.Context, error)
 	if err != nil {
 		return nil, fmt.Errorf("unable to contact server: %v", err)
 	}
-	rc := &remoteContext{
+	return &Client{
 		client: client,
 		url:    u,
-	}
-	ctx := internal.WithCallOverride(context.Background(), rc.call)
-	ctx = internal.WithLogOverride(ctx, rc.logf)
-	ctx = internal.WithAppIDOverride(ctx, appID)
+		appID:  appID,
+	}, nil
+}
+
+// NewContext returns a copy of parent that will cause App Engine API
+// calls to be sent to the client's remote host.
+func (c *Client) NewContext(parent context.Context) context.Context {
+	ctx := internal.WithCallOverride(parent, c.call)
```

Comment:
call, logf, appID ... are methods of remoteContext and need to be changed to of Client.

Reply:
Whoops, ran `go test` in the wrong terminal window. Done.